.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
   - npm install -g bower
   - npm install --prefix ./frontend
   - npm install --prefix ./backend
-  - cd ./frontend && bower install
+  - cd ./frontend && bower install && cd ..
```

Comment:
Folder changing could be avoided with the .bowerrc file.
[Bower .bowerrc Config](https://bower.io/docs/config/)